AUTHORS

```diff
@@ -0,0 +1,9 @@
+# This is the list of CloudEvents authors for copyright purposes.
```

Comment:
Do we must have this file? I would prefer if we don't, because it needs to be kept updated

Reply:
I think it is used to define what "CloudEvents Authors" mean. Whenever I see a group name in the copyright, I always see an authors file.

Reply:
I could be wrong... I am not a lawyer...

Reply:
if that's the case, this doesn't represent all the authors, because there are people that have contributed to this project outside these 4 companies. I think the git commits suffice to define "cloudevents authors", but yeah i'm not a lawyer too :smile:

Reply:
I am gonna drop it...